Reply/app/src/main/java/com/example/reply/ui/ReplyListContent.kt

```diff
@@ -69,20 +75,31 @@ fun ReplyInboxScreen(
         }
     }
 
+    var selectedEmailIds by remember { mutableStateOf(setOf<Long>()) }
+    val swapEmailSelection = { id: Long ->
+        if (selectedEmailIds.contains(id))
+            selectedEmailIds -= id
+        else selectedEmailIds += id
+    }
```

Comment:
Could this form part of `ReplyHomeUIState`?

Reply:
Done!